packages/web-components/src/components/quote/__stories__/quote.stories.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { select } from '@storybook/addon-knobs';
+import { html } from 'lit-element';
+import '../quote';
+import '../../link-with-icon/link-with-icon';
+import ArrowRight20 from 'carbon-web-components/es/icons/arrow--right/20.js';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+
+import readme from './README.stories.mdx';
+
+export const Default = ({ parameters }) => {
+  const { copy, quoteMark, sourceHeading, sourceCopy, sourceBottomCopy } = parameters?.props?.Quote ?? {};
+  return html`
+    <dds-quote inverse="false" markType="${quoteMark}">
+      <span slot="copy">
+        ${copy}
+      </span>
+      <span slot="sourceHeading">
+        ${sourceHeading}
+      </span>
+      <span slot="sourceCopy">
+        ${sourceCopy}
+      </span>
+      <span slot="sourceBottomCopy">
+        ${sourceBottomCopy}
+      </span>
+      <dds-link-with-icon slot="cta" href="https://example.com">
+        Link with Icon ${ArrowRight20({ slot: 'icon' })}
+      </dds-link-with-icon>
+    </dds-quote>
+  `;
+};
+
+const types = {
+  singleCurved: 'singleCurved',
+  doubleCurved: 'doubleCurved',
+  doubleAngle: 'doubleAngle',
+  singleAngle: 'singleAngle',
+  lowHighReversedDoubleCurved: 'lowHighReversedDoubleCurved',
+};
```

Comment:
Good to try bringing the list from enum - Here's an example: https://github.com/carbon-design-system/ibm-dotcom-library/blob/v1.11.0/packages/web-components/src/components/button/__stories__/button.stories.ts#L22-L27